src/graphics/env-lighting.js

```diff
@@ -10,6 +10,7 @@ import {
 import { DebugGraphics } from './debug-graphics.js';
 
 const fixCubemapSeams = true;
+const RGBA8_TYPE = TEXTURETYPE_RGBM;
```

Comment:
- local variables should not be all capital in `RGBA8_TYPE`
- should this default to RGBP now?

Reply:
ah this actually is a global constant .. ignore the first point :)